src/webui/www/private/scripts/prop-general.js

```diff
@@ -98,14 +98,15 @@ window.qBittorrent.PropGeneral = (function() {
                     let temp;
                     // Update Torrent data
                     if (data.seeding_time > 0)
-                        temp = "QBT_TR(%1 (%2 this session))QBT_TR[CONTEXT=PropertiesWidget]"
+                        temp = "QBT_TR(%1 (seeded for %2))QBT_TR[CONTEXT=PropertiesWidget]"
                         .replace("%1", window.qBittorrent.Misc.friendlyDuration(data.time_elapsed))
                         .replace("%2", window.qBittorrent.Misc.friendlyDuration(data.seeding_time));
                     else
                         temp = window.qBittorrent.Misc.friendlyDuration(data.time_elapsed);
                     $('time_elapsed').set('html', temp);
 
-                    $('eta').set('html', window.qBittorrent.Misc.friendlyDuration(data.eta));
+                    const MAX_ETA = 8640000; // 100 days
```

Comment:
`MAX_ETA` should be defined as a const variable in `window.qBittorrent.Misc`

Reply:
Sorry i don't know javascript. I don't suppose it would be as simple as adding `const MAX_ETA: 8640000` in...?
https://github.com/qbittorrent/qBittorrent/blob/a8ae97ba917259ec91868553aa881e4c0f0cd701/src/webui/www/private/scripts/misc.js#L37

Reply:
>I don't suppose it would be as simple as adding const MAX_ETA: 8640000 in...?

It is :)
```js
window.qBittorrent.Misc = (function() {
    const exports = function() {
        return {
            friendlyUnit: friendlyUnit,
            friendlyDuration: friendlyDuration,
            friendlyPercentage: friendlyPercentage,
            friendlyFloat: friendlyFloat,
            parseHtmlLinks: parseHtmlLinks,
            escapeHtml: escapeHtml,
            safeTrim: safeTrim,
            toFixedPointString: toFixedPointString,
            containsAllTerms: containsAllTerms,
            MAX_ETA: 8640000
        };
    };
    // ...
};

// use it
window.qBittorrent.Misc.MAX_ETA
```

Reply:
But then it won't be const. I can 
`window.qBittorrent.Misc.MAX_ETA = x;`
before i call it.

Reply:
You can, but the value won't retain IIRC:
```js
window.qBittorrent.Misc.MAX_ETA = 0;
const isZero = window.qBittorrent.Misc.MAX_ETA == 0; // false
```

Reply:
>but the value won't retain IIRC

It does actually, i just tested it.

Reply:
Still mutable.

Reply:
mutable but it should retain the original value.

Reply:
I don't know enough to debate so i'll just change it.
Maybe it's not the right way to test so i'll mention what i do: i do the following and every torrent's eta column shows 10.
```
--- a/src/webui/www/private/scripts/dynamicTable.js
+++ b/src/webui/www/private/scripts/dynamicTable.js
@@ -1104,7 +1104,7 @@ window.qBittorrent.DynamicTable = (function() {
               // eta
               this.columns['eta'].updateTd = function(td, row) {
-              const MAX_ETA = 8640000; // 100 days
-              const eta = window.qBittorrent.Misc.friendlyDuration(this.getRowValue(row), MAX_ETA);
+              window.qBittorrent.Misc.MAX_ETA = 10;
+              const eta = window.qBittorrent.Misc.MAX_ETA;
               td.set('text', eta);
               td.set('title', eta);    
           );
```

Last question: am i supposed to write something where you wrote `// ...`?

Reply:
@thalieht 

Why not increase `MAX_ETA` to something like 50 years? Many users seed many things way beyond 100 days and would surely appreciate being able to see more exact estimations in these cases. It comes at no additional cost to us since the friendly duration function already handles `years`.

Of course handling leap years or whatever is still out-of-scope.

Reply:
I believe 100 days was chosen arbitrarily a long time ago (there was no seeding time limit then) and 100 days is the default in GUI as well. Because of that, i can't change it in this PR. I think there is an issue about this (seeding time and ETA >100 days).
It's an easy change but i don't care enough to "risk" opposition if i were to make a PR.

Reply:
@thalieht I see, it is indeed a good idea to defer such a change to a PR that also handles the GUI case.

Reply:
>Maybe it's not the right way to test so i'll mention what i do: i do the following and every torrent's eta column shows 10.

Your testing is valid but the result is unexpected... anyway please push your code and we will see what to do.

>Last question: am i supposed to write something where you wrote // ...?

No need.